spec/requests/entry_requests_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe "EntryRequests", type: :request do
+  let(:entry_request) { create(:entry_request) }
+  let(:evaluation) { entry_request.evaluation }
+  let(:group) { evaluation.group }
+  let(:selected_user) { entry_request.user }
+  let(:arguments) do
+    ["/groups/#{group.id}/evaluations/#{evaluation.id}/entryrequests/update",
+     params: {
+         user_id:    selected_user.id,
+         entry_type: EntryRequest::AB
+     }]
+  end
```

Comment:
DRY-ing up the test is great, but in this case I think it's too much. I constantly have to scroll to-and-fro to get what happens.

Reply:
Yes, in this case it could be unnecessary.